web-registry/src/components/templates/ProjectDetails/hooks/useSeo.ts

```diff
@@ -33,11 +30,6 @@ export default function useSeo({
   const metadataLocation = metadata?.['schema:location'];
   const projectAddress = metadataLocation?.['place_name'];
 
-  useEffect(() => {
-    setPageView(location);
```

Comment:
this code is unnecessary with the new page view configuration. https://github.com/regen-network/regen-web/pull/1188/files#r988433970